manifests/config.pp

```diff
@@ -228,6 +228,19 @@
         content => template('foreman/settings-external-auth.yaml.erb'),
         order   => '02',
       }
+
+      include apache::mod::auth_basic
```

Comment:
Doesn't this belong inside the `if` statement, given it's needed for the apache fragment?

Reply:
I've removed that condition since the whole block is already under the same `if` statement.